rotkehlchen/inquirer.py

```diff
@@ -648,7 +668,24 @@ def _find_usd_price(
                 price, oracle = Inquirer._query_fiat_pair(base=asset, quote=Inquirer.usd)
                 return price, oracle, False
 
-        # continue, asset isn't fiat or a price can be found by one of the oracles (CC for example)
+        # continue, asset isn't fiat, check manual prices
+        if GlobalDBHandler.get_manual_current_price(asset=asset) is not None:
+            price, used_main_currency, _ = Inquirer._try_oracle_price_query(
+                oracle_instance=Inquirer._manualcurrent,
+                from_asset=asset,
+                to_asset=A_USD,
+                coming_from_latest_price=coming_from_latest_price,
+                match_main_currency=match_main_currency,
+            )
+            Inquirer.set_cached_price(cache_key=cache_key, cached_price=CachedPriceEntry(
+                price=price,
+                time=ts_now(),
+                oracle=CurrentPriceOracle.MANUALCURRENT,
+                used_main_currency=used_main_currency,
+            ))
```

Comment:
can we add this to `_try_oracle_price_query`? If you check the other function we check if it is zero and if it is not then we log and add to the cache